modpack_format/modpack_example.modip.json

```diff
@@ -0,0 +1,35 @@
+{
+  "formatType": "modipModpack",
+  "formatVersion": "1.0.0",
+  "id": "modpack-example",
+  "name": "Modpack Example",
+  "summary": "Summary of the pack",
+  "description": "Detailed description of the modpack. <b>HTML?</b>",
+  
+  "updates": "https://api.example.com/pack/modpack-example.modip.json",
+
+  "dependencies": [
+    {
+      "id": "minecraft",
+      "version": "1.16.4"
```

Comment:
Do the IDs mean anything across different modpacks?

Reply:
Probably not, but I'm not sure.

Reply:
Apart from special values, what is it used for?

Reply:
It's there for an easy way to refer to dependencies both in code and in messages. For example if a download failed, a launcher can display "Downloading `mymod` failed..." instead of "Downloading dependency 3" failed, which is much clearer. It helps clear things up. Other than that though there is not much use I can think of off the top of my head. I put it there because I just felt it "should be there" to be honest.